sdk/tables/azure-data-tables/samples/sample_query_table.py

```diff
@@ -68,59 +68,27 @@ def sample_query_entities(self):
         from azure.data.tables import TableClient
         from azure.core.exceptions import HttpResponseError
 
-        print("Entities with name: marker")
         # [START query_entities]
         with TableClient.from_connection_string(self.connection_string, self.table_name) as table_client:
             try:
-                parameters = {"name": "marker"}
-                name_filter = "Name eq @name"
-                queried_entities = table_client.query_entities(
-                    query_filter=name_filter, select=["Brand", "Color"], parameters=parameters
+                print("Entities with name: marker and brand: Crayola")
+                queried_entities1 = table_client.query_entities(
+                    query_filter={"name": "marker", "brand": "Crayola"},
+                    select=["Brand", "Color"],
+                    parameters="Name eq @name and Brand eq @brand",
+                    headers ={"Accept" : "application/json;odata=nometadata"}
```

Comment:
We should not add this to an existing sample - let's leave the existing samples untouched as adding this header should be very uncommon.
I would add a new method called something like `sample_query_entities_without_metadata` and add it the bottom of the file.

Reply:
Updated, added a new function for `no metadata` headers.